### DISCUSSION TITLE
How to use coredump?

### DISCUSSION THREAD

**Discussion by hunter-nl** [dingo35/SmartEVSE-3.5#67] [Q&A] [answered]
In the partition table a coredump partition is added. How can I make use of this as developer? I got a crash of esp32. But how to get that coredump and process this to get something readable to understand why it was crashed (at which point in the code)?

**Comment by dingo35** [comment, marked as answer]
I just added coredumping.txt to the repo with all the instructions.

See d76951e062

**Reply by hunter-nl** [reply]
Thanks!
